### PR TITLE
Set up CI with Azure Pipelines

### PR DESCRIPTION
**Pull request by johnbuonassisi**
This is a description for the PR

Testing 123

More infor added!!!

### CODE REVIEW

**Comment by johnbuonassisi** [comment]
Left a comment 😃

**Comment by johnbuonassisi** [comment]
Test leave a comment

**Comment by johnbuonassisi** [comment]
1 comment

**Comment by johnbuonassisi** [comment]
2 comment

**Comment by ghost** [comment]
Hello

**Comment by ghost** [comment]
Hi there!

**Comment by ghost** [comment]
Hi there, 2!

**Comment by ghost** [comment]
One more time!

**Comment by ghost** [comment]
Again!

**Comment by johnbuonassisi** [comment]
A comment!

**Comment by johnbuonassisi** [comment]
Hello!

**Comment by johnbuonassisi** [comment]
Test Comment

File: azure-pipelines.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# Go
+# Build your Go project.
+# Add steps that test, save build artifacts, deploy, and more:
+# https://docs.microsoft.com/azure/devops/pipelines/languages/go
+
+trigger:
+- master
```
Comment (johnbuonassisi) [inline azure-pipelines.yml]: test

**Review by johnbuonassisi — COMMENTED** [review]
test1

**Comment by johnbuonassisi** [comment]
test

**Comment by johnbuonassisi** [comment]
OVerall

File: azure-pipelines.yml (hunk 1 above)
Comment (johnbuonassisi) [reply azure-pipelines.yml]: help!

Comment (johnbuonassisi) [reply azure-pipelines.yml]: help again!

File: azure-pipelines.yml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+steps:
+- script: |
+    mkdir -p '$(GOBIN)'
+    mkdir -p '$(GOPATH)/pkg'
+    mkdir -p '$(modulePath)'
+    shopt -s extglob
+    shopt -s dotglob
+    mv !(gopath) '$(modulePath)'
+    echo '##vso[task.prependpath]$(GOBIN)'
+    echo '##vso[task.prependpath]$(GOROOT)/bin'
```
Comment (johnbuonassisi) [inline azure-pipelines.yml]: test!

**Comment by johnbuonassisi** [comment]
great!

**Comment by johnbuonassisi** [comment]
Test

File: azure-pipelines.yml [hunk 3]
```diff
@@ -0,0 +1,39 @@
+# Go
```
Comment (johnbuonassisi) [inline azure-pipelines.yml]: test comment

File: azure-pipelines.yml [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+# https://docs.microsoft.com/azure/devops/pipelines/languages/go
+
+trigger:
+- master
+
+pool:
+  vmImage: 'ubuntu-latest'
+
+variables:
+  GOBIN:  '$(GOPATH)/bin' # Go binaries path
```
Comment (johnbuonassisi) [inline azure-pipelines.yml]: comment

**Comment by johnbuonassisi** [comment]
Add a Issue Comment.

File: azure-pipelines.yml [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+  GOROOT: '/usr/local/go1.11' # Go installation path
+  GOPATH: '$(system.defaultWorkingDirectory)/gopath' # Go workspace path
+  modulePath: '$(GOPATH)/src/github.com/$(build.repository.name)' # Path to the module's code
+
+steps:
+- script: |
+    mkdir -p '$(GOBIN)'
+    mkdir -p '$(GOPATH)/pkg'
```
Comment (johnbuonassisi) [inline azure-pipelines.yml]: A Pull request comment

File: azure-pipelines.yml [hunk 6]
```diff
@@ -0,0 +1,39 @@
+# Go
+# Build your Go project.
+# Add steps that test, save build artifacts, deploy, and more:
```
Comment (ghost) [inline azure-pipelines.yml]: What is this?

**Comment by johnbuonassisi** [comment]
Test comment

File: azure-pipelines.yml [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+  modulePath: '$(GOPATH)/src/github.com/$(build.repository.name)' # Path to the module's code
+
+steps:
+- script: |
+    mkdir -p '$(GOBIN)'
+    mkdir -p '$(GOPATH)/pkg'
+    mkdir -p '$(modulePath)'
+    shopt -s extglob
+    shopt -s dotglob
+    mv !(gopath) '$(modulePath)'
```
Comment (johnbuonassisi) [inline azure-pipelines.yml]: what the heck

**Comment by ghost** [comment]
Another user test

File: azure-pipelines.yml [hunk 8]
```diff
@@ -0,0 +1,39 @@
...
+  GOROOT: '/usr/local/go1.11' # Go installation path
+  GOPATH: '$(system.defaultWorkingDirectory)/gopath' # Go workspace path
+  modulePath: '$(GOPATH)/src/github.com/$(build.repository.name)' # Path to the module's code
+
+steps:
+- script: |
+    mkdir -p '$(GOBIN)'
+    mkdir -p '$(GOPATH)/pkg'
+    mkdir -p '$(modulePath)'
+    shopt -s extglob
```
Comment (johnbuonassisi) [inline azure-pipelines.yml]: Hello

**Comment by ghost** [comment]
Hello!

**Comment by johnbuonassisi** [comment]
Test high level comment

**Pull request closed without merging**